src/app/Library/CrudPanel/Traits/Tabs.php

```diff
@@ -106,56 +106,77 @@ public function isLastTab($label)
     }
 
     /**
+     * @deprecated Do not use this method as it will be removed in future versions!
+     * Instead, use $this->getElementsWithoutATab($this->getCurrentFields())
+     *
      * @return \Illuminate\Support\Collection
      */
     public function getFieldsWithoutATab()
     {
-        $all_fields = $this->getCurrentFields();
+        return $this->getElementsWithoutATab($this->getCurrentFields());
+    }
 
-        $fields_without_a_tab = collect($all_fields)->filter(function ($value, $key) {
+    /**
+     * @return \Illuminate\Support\Collection
+     */
+    public function getElementsWithoutATab(array $elements)
+    {
+        return collect($elements)->filter(function ($value) {
             return ! isset($value['tab']);
         });
+    }
 
-        return $fields_without_a_tab;
+    /**
+     * @return array|\Illuminate\Support\Collection
+     */
+    public function getTabFields(string $label)
+    {
+        return $this->getTabElements($label, $this->getCurrentFields());
     }
 
     /**
-     * @param $label
      * @return array|\Illuminate\Support\Collection
      */
-    public function getTabFields($label)
+    public function getTabColumns(string $label)
     {
-        if ($this->tabExists($label)) {
-            $all_fields = $this->getCurrentFields();
+        return $this->getTabElements($label, $this->columns());
+    }
 
-            $fields_for_current_tab = collect($all_fields)->filter(function ($value, $key) use ($label) {
+    /**
+     * @return array|\Illuminate\Support\Collection
+     */
+    public function getTabElements(string $label, array $elements)
+    {
+        if ($this->tabExists($label)) {
+            return collect($elements)->filter(function ($value) use ($label) {
                 return isset($value['tab']) && $value['tab'] == $label;
             });
-
-            return $fields_for_current_tab;
         }
 
         return [];
     }
 
-    /**
-     * @return array
-     */
-    public function getTabs()
+    public function getTabs(): array
```

Comment:
Pedro and I have decided to 

- [ ] create a NEW method that does the same thing, but applies to both fields and columns:

```php
    public function getUniqueTabNames($source): array
    // $source could be `fields` or `columns` for now
```

- [ ] make `getTabs()` call `getUniqueTabNames('fields')` so that there are no breaking changes inside it, and we keep the method as-is, with the same signature;